spec/fixtures/state_file/fed_return_xmls/2023/id/miranda_1099r.xml

```diff
@@ -153,7 +153,7 @@
             <W2StateLocalTaxGrp>
                 <W2StateTaxGrp>
                     <StateAbbreviationCd>ID</StateAbbreviationCd>
-                    <EmployerStateIdNum>00-0000005</EmployerStateIdNum>
+                    <EmployerStateIdNum>000000005</EmployerStateIdNum>
```

Comment:
w2 sync was failing for this fixture b/c `-` was included and w2 `employer_ein` was failing validation

>   validates :employer_state_id_num, format: { with: /\A(\d{0,17})\z/, message: ->(_object, _data) { I18n.t('state_file.questions.w2.edit.employer_state_id_error') } }